docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - ~/.lein:/home/akvo/.lein
       - ./postgres/provision:/pg-certs
    links:
-      - keycloak:auth.lumen.localhost
```

Comment:
Do this changes in the hostname require a change in the documentation? I can think of README.dev.md at least.